src/py42/clients/settings/device_settings.py

```diff
@@ -264,6 +284,13 @@ def _build_regex_exclusions(self):
         }
         self.data[u"backupPaths"][u"excludeUser"] = user_exclude_dict
 
+    @property
+    def locked(self):
+        """Indicates whether the backup set as a whole is locked. If True, individual
+        settings for this backup set (except for Destination settings), cannot be modified.
+        """
+        return u"@locked" in self.data and self.data[u"@locked"] == u"true"
```

Comment:
To be safe, it might be better to call your `str_to_bool()` method here and make sure the result of that call is `True`.

Reply:
`and str_to_bool(self.data[u"@locked"])`